dicom_numpy/zip_archive.py

```diff
@@ -2,7 +2,10 @@
 import logging
 import tempfile
 
-import dicom
+try:
+    import pydicom as dicom
+except ImportError:
+    import dicom
```

Comment:
I'm glad pydicom 1.0 support was this easy to add for us!

Reply:
It was surprisingly cut-and-dry